dao/src/main/java/greencity/repository/BagRepository.java

```diff
@@ -64,4 +64,16 @@ public interface BagRepository extends JpaRepository<Bag, Integer> {
         + "JOIN order_bag_mapping obm on b.id = obm.bag_id "
         + "WHERE obm.ORDER_ID = :orderId", nativeQuery = true)
     List<Map<String, Object>> getBagInfo(Long orderId);
+
+    /**
+     * method, that returns {@link List}of{@link Bag}.
+     *
+     * @param orderId order id
+     * @return {@link List}of{@link Bag} by orderId.
+     * @author Max Boiarchuk
```

Comment:
please change auhor

Reply:
Who is the author of this code? `Max Boiarchuk` or @JoseCaste?

Reply:
Max Boiarchuk, but my name appears maybe because I run formatter and checkstyle to PR :o

Reply:
But I changed the table name to lower case, following the standar but I see that they are in upper case

Reply:
@JoseCaste I asked about it 
` * @author Max Boiarchuk`
I see that this code is added by you but in the java doc I see that author is  Max Boiarchuk